Source/WebCore/dom/Element.cpp

```diff
@@ -2466,32 +2466,40 @@ void Element::invalidateEventListenerRegions()
     invalidateStyleInternal();
 }
 
-bool Element::hasDisplayContents() const
+bool Element::hasDisplayType(DisplayType displayType) const
```

Comment:
I realize this is private but it should be clear in its name it only works for stored styles, like hasDisplayForStoredStyle(). Not sure a helper is even needed really.

Reply:
I was adding it to avoid some code duplication, but it's also not exactly rocket science, so I might remove it altogether.